evaluation/EDA/run_infer.py

```diff
@@ -185,6 +186,13 @@ def process_instance(instance, agent_class, metadata, reset_logger: bool = True)
         type=str,
         help='data split, eg, test',
     )
+    parser.add_argument(
+        '-gmi',
+        '--global-max-iterations',
+        type=int,
+        default=config.global_max_iterations,
+        help='The maximum number of iterations to run the agent globally',
```

Comment:
Let's remove these arguments in evaluation folders. None of those evaluation leverages agent delegation so this extra arg doesn't add value.

Reply:
So comparing iterations in these files have no significance?